layers/meta-balena-raspberrypi/recipes-bsp/bootfiles/rpi-config_git.bbappend

```diff
@@ -33,7 +33,13 @@ do_deploy:append:revpi-connect-s() {
 	echo "dtoverlay=revpi-connect" >> ${DEPLOYDIR}/bootfiles/config.txt
         echo "dtoverlay=dwc2" >> ${DEPLOYDIR}/bootfiles/config.txt
         echo "dr_mode=host" >> ${DEPLOYDIR}/bootfiles/config.txt
+}
 
+do_deploy:append:revpi-connect-4() {
+	# Use the RevPi Connect device tree overlay
+	echo "dtoverlay=revpi-connect" >> ${DEPLOYDIR}/bootfiles/config.txt
+        echo "dtoverlay=dwc2" >> ${DEPLOYDIR}/bootfiles/config.txt
+        echo "dr_mode=host" >> ${DEPLOYDIR}/bootfiles/config.txt
```

Comment:
Not needed on Connect 4. It has a real XHCI controller and does not use the integrated on compute module